tools/fv_nudge.F90

```diff
@@ -139,6 +138,7 @@ module fv_nwp_nudge_mod
 
  public fv_nwp_nudge, fv_nwp_nudge_init, fv_nwp_nudge_end, breed_slp_inline, T_is_Tv
  public do_adiabatic_init
+ public nwp_nudge_int
```

Comment:
@bensonr I see that there are some new options in this file. Was this to merge in the several different versions of the nudging? (If so, good job :-)

Reply:
@lharris4 - the new options in this file are the result of work done by @wfcooke to add in updates from the SPEAR needs.  The other nudging versions are still in separate source files.